### PR TITLE
Remove or Fix Unused Variables

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/datetime.d [hunk 1]
```diff
@@ -29838,9 +29836,9 @@ public:
                     inUTC = readVal!bool(tzFile);
             }
 
-            _enforceValidTZFile(tzFile.readln().strip().empty);
```
Comment (DmitryOlshansky) [inline std/datetime.d]: This one is suspicious - it does read a line twice, so it might be intended

Comment (JackStouffer) [inline std/datetime.d]: Let's wait for the auto tester. If it fails I'll just delete the variable

**Comment by quickfur** [comment]
LGTM.

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Pull request merged**